### ISSUE TITLE
 The Role as 'link' or suggestion as 'press enter to select' is not announced for the controls present in the table of 'Design' page.

### ISSUE THREAD

**Issue by msftedad** [angular-ui/ui-grid#7236]
### Description: ###
Screen reader users will get impacted as the user will not be able to know whether the searched keyword is valid or invalid and cannot access the control efficiently if the screen reader is not announcing any information about the search result after searching any invalid keyword.

### Test Environment: ###
OS Build: Version 21H2(22000.613)
Browser: Chrome Version 100.0.4896.127 (Official Build) (64-bit) 
Screen reader: Non-Visual Desktop Access (NVDA) (2021.1)

### Repro steps: ###
1. Open [Angular Grid | Excel Style Filtering | Infragistics](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel-style-filtering)
2. Grids excel style filtering pen page will open.
3. Reach to the main landmark content where ' Examples' tab is selected.
4. With 6-tab press, reach to the column header of the available table.
5. Navigate on the table cells.
6. Verify the issue that the role for the interactive table rows is announced or not.

### Actual Results: ###
The Role of 'link' or suggestion as 'press enter to select' is not announced for the controls present in the table of the 'Design' page.
NVDA is announcing as: "01 design initiation name"
O1 design initiation column 1 of 3
01 design initiation selected name-jaws.

### Expected results: ###
The Role of 'link' or suggestion as 'press enter to select' should be announced for the controls present in the table of the 'Design' page.
NVDA should announce as "01 design initiation name button/link".

### Attachments: ###
<img width="958" alt="MAS4 1 2_role not announced_" src="https://user-images.githubusercontent.com/93735775/183306823-15b6806c-d497-41d4-91f0-9e6f497420e7.png">

### Additional information: ###
Same issue Repro with Narrator and JAWS.
O1 design initiation column 1 of 3 -Narrator Behavior
01 design initiation selected name- JAWS Behavior
Same Issue Reproes with all the controls present in the tables throughout the application

**Comment by mportuga** [comment]
This defect is not for UI-Grid, but for Ignite UI, which is managed by a completely different team.
